roles/dnsmasq/templates/cifmw-dnsmasq.conf.j2

```diff
@@ -1,17 +1,16 @@
-# Disable DNS
-port=0
-# Avoid conflict with other DNS listening on lo
-except-interface=lo
+# Managed by ci-framework/dnsmasq
+
 user=dnsmasq
 group=dnsmasq
 pid-file=/var/run/cifmw-dnsmasq.pid
 dhcp-leasefile=/var/lib/dnsmasq/cifmw-dnsmasq.leases
-{% for key,val in cifmw_dnsmasq_global_options.items() %}
-{%   if val is not none                                %}
+
+{% for key,val in cifmw_dnsmasq_global_options.items() -%}
+{%-   if val is not none                               -%}
```

Comment:
so @hjensas thought it might be nicer to use have a list of strings?
that way, we could pass anything we like:
```YAML
cifmw_dnsmasq_global_options:
  - 'port=0'
  - log-queries
  - exclude-interface=lo
  - exclude-interface=nat64-v4
  - exclude-interface=nat64-v6
```
and the like. That would allow to chose between both supported syntax (coma-separated values, or multiple option occurrences) in an easy way.

Thoughts?